src/App.tsx

```diff
@@ -1,61 +1,122 @@
 import './App.scss';
+import { FC, useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { prepareTodos, getNewTodoId, getUserById } from './helpers';
+import { TodoWithUser } from './types/types';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const preparedTodos = prepareTodos(todosFromServer, usersFromServer);
+
+export const App: FC = () => {
+  const [userSelect, setUserSelect] = useState(0);
+  const [todos, setTodos] = useState(preparedTodos);
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+  const [usersError, setUsersError] = useState(false);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!title) {
```

Comment:
I can push todo with spaces in title

![image](https://github.com/mate-academy/react_add-todo-form/assets/84296041/bf611a43-f49d-4fce-90f6-e10a9c3902e0)